src/core/react-query/group/queries.ts

```diff
@@ -34,9 +34,10 @@ export const useGroupsInfiniteQuery = (params: GroupsInfiniteRequestType) =>
     },
   });
 
-export const useGroupSeriesQuery = (groupId: number, enabled = true) =>
+export const useGroupSeriesQuery = ({ enabled = true, groupId, sorted = false }: GroupSeriesRequestType) =>
   useQuery<SeriesType[]>({
     queryKey: ['group-series', groupId],
     queryFn: () => axios.get(`Group/${groupId}/Series`),
+    select: data => (sorted ? data.sort((a, b) => (a.IDs.ID > b.IDs.ID ? 1 : -1)) : data),
```

Comment:
Why not sort them in the component itself with a useMemo?

Reply:
Already resolved as requested! Had a moment and completely forgot that this would impact anything that tried using this query...